.github/actions/bot/action.yml

```diff
@@ -24,255 +25,38 @@ runs:
-          || (echo "TIMEOUT" && exit 1)
-
-        # comment with the response
-        if [[ -f response.md ]]; then
-          gh issue comment ${{ github.event.issue.number }} -R ${{ github.repository }} --body-file=response.md
-        else
-          echo "No response was generated."
-        fi
-
-        # stage changes
-        git add -A
-
-    - name: Commit, push, comment
-      if: steps.detect_command.outputs.gptme_command && steps.determine_action.outputs.action_type == 'make_changes'
-      shell: bash
-      env:
-        GITHUB_TOKEN: ${{ inputs.github_token }}
-      run: |
-        # generate commit message
-        docker run --rm \
-          -v ~/.config/gptme:/home/appuser/.config/gptme \
-          -v $GITHUB_WORKSPACE:/workspace \
-          -w /workspace \
-          ghcr.io/erikbjare/gptme:latest \
-          --non-interactive \
-          "Run 'git diff --staged' to inspect what has changed." \
-          "-" "Write a commit message for it to 'message.txt'. Use the 'conventional commits' style."
-
-        # Read and format log
-        ./scripts/format_log.sh ~/.local/share/gptme/logs/*/conversation.jsonl > log.txt
-
-        RUN_URL="https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}"
-        COMMENT_URL="https://github.com/${{ github.repository }}/issues/${{ github.event.issue.number }}#issuecomment-${{ github.event.comment.id }}"
-
-        # commit message & description
-        COMMIT_MSG="$(cat message.txt || echo 'no commit message')"
-        COMMIT_DESC="\`gptme '${{ steps.detect_command.outputs.gptme_command }}'\`
-
-        Triggered by: $COMMENT_URL
-        Run: $RUN_URL"
-
-        # commit message with description
-        COMMIT_MSG_FULL="$COMMIT_MSG
-
-        $COMMIT_DESC"
-
-        # commit message with description and log
-        COMMIT_MSG_FULL_WITH_LOG="$COMMIT_MSG_FULL
-
-        <details>
-        <summary>Log</summary>
-        <pre>$(cat log.txt || echo 'could not get log')</pre>
-        </details>"
-
-        git config user.name "gptme-bot"
-        git config user.email "gptme-bot@superuserlabs.org"
-
-        git commit -m "$COMMIT_MSG_FULL"
-
-        git push -u origin ${{ steps.checkout_branch.outputs.branch_name }}
-
-        if [[ ${{ github.event.issue.pull_request != null }} == "true" ]]; then
-          echo "Changes have been pushed to this pull request." | gh pr comment ${{ github.event.issue.number }} -R ${{ github.repository }} --body-file=-
-        else
-          # Some say this helps! https://github.com/cli/cli/issues/2691#issuecomment-1289521962
-          sleep 1
-          PR_URL=$(gh pr create --title "$COMMIT_MSG" --body "$COMMIT_MSG_FULL_WITH_LOG" --repo ${{ github.repository }} | grep -o 'https://github.com[^ ]*')
-          echo "A pull request has been created for this issue: $PR_URL" | gh issue comment ${{ github.event.issue.number }} -R ${{ github.repository }} --body-file=-
-        fi
+        python scripts/github_bot.py
```

Comment:
Will this work for other repos using this bot action given that it's a reusable workflow?